lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -1221,6 +1221,26 @@ declare class Complex64Array implements Complex64ArrayInterface {
 	*/
 	subarray( begin?: number, end?: number ): Complex64Array;
 
+	/**
+	* Returns a string representating the elements of a typed array.
+	*
+	* @param locales - a locale identifier
+	* @param options - an object containing other options
+	* @throws locales argument must be a string
+	* @throws options argument must be an object
```

Comment:
```suggestion
```

These sorts of annotations are not necessary in this case. A TypeScript is strictly typed, a user cannot provide an invalid variable having a non-specified type. In which case, these "throws" annotations cannot happen and are not useful to the reader.